src/main/java/mil/dds/anet/database/mappers/PersonMapper.java

```diff
@@ -41,12 +39,7 @@ public Person map(ResultSet rs, StatementContext ctx) throws SQLException {
 		a.setPhoneNumber(r.getString("people_phoneNumber"));
 		a.setCountry(r.getString("people_country"));
 		a.setGender(r.getString("people_gender"));
-		
-		Timestamp endOfTourDate = r.getTimestamp("people_endOfTourDate");
-		if (endOfTourDate != null) { 
-			a.setEndOfTourDate(new DateTime(endOfTourDate));
-		}
-		
+		a.setCreatedAt(DaoUtils.getInstantAsLocalDateTime(r, "people_endOfTourDate"));
```

Comment:
Here you should use setEndOfTourDate instead of setCreatedAt.

Reply:
Ouch!